scripts/build/binary

```diff
@@ -56,18 +57,19 @@ fi
 
 if [ "$(go env GOOS)" = "windows" ]; then
     # Generate a Windows file version of the form major,minor,patch,build
-    VERSION_QUAD=$(printf "%s" "$VERSION" | sed -re 's/^([0-9.]*).*$/\1/' | tr . , | sed -re 's/^[0-9]+$/\0,0/' | sed -re 's/^[0-9]+,[0-9]+$/\0,0/' | sed -re 's/^[0-9]+,[0-9]+,[0-9]+$/\0,0/')
+    VERSION_QUAD=$(printf "%s" "$VERSION" | sed -re 's/^([0-9.]*).*$/\1/' | tr . , | sed -re 's/^([0-9]+),?$/\1,0/' | sed -re 's/^([0-9]+,[0-9]+),?$/\1,0/' | sed -re 's/^([0-9]+,[0-9]+,[0-9]+),?$/\1,0/')
```

Comment:
don't you just want to remove the `,` if it is last char?

Reply:
I guess that works too. Let me try.